docs/Configuration.md

```diff
@@ -162,6 +162,27 @@ resolutions:
 For details of the available reasons, see
 [RuleViolationResolutionReason.kt](../model/src/main/kotlin/config/RuleViolationResolutionReason.kt)
 
+### Curations
+
+#### License Findings
```

Comment:
Rather use `### Curating license findings` for consistency with the wording in other headers.

Reply:
Done.